solutions/04_grade/grade.js

```diff
@@ -0,0 +1,19 @@
+var grade = 100;
+
+if (grade <= 65)
+{
+    console.log("F");
+}
+else if(grade <= 75)
+{
+    console.log("C");
+}
+
+else if(grade <= 85);
+{
+    console.log("B")
+}
+else
+{
+    console.log("A")
+}
```

Comment:
This is great!

Just make sure to *not* follow `if` statements with semicolon (`else if(grade <= 85);` on line 5), and *do* add semicolons after expressions like `console.log()` (lines 14 and 18)